parley/src/layout/data.rs

```diff
@@ -100,6 +99,11 @@ impl ClusterInfo {
     pub(crate) fn is_whitespace(&self) -> bool {
         self.source_char.is_whitespace()
     }
+
+    #[cfg(test)]
+    pub(crate) fn source_char(&self) -> char {
```

Comment:
I'd really like to test this in `check_cluster_snapshot`, so I've exposed it for tests only.